gateway/gateway.go

```diff
@@ -284,13 +356,37 @@ func (gateway *HandleT) webRequestBatchDBWriter(process int) {
 	}
 }
 
-func (gateway *HandleT) dedupWithBadger(body *[]byte, messageIDs [][]byte, writeKey string, writeKeyDupStats map[string]int) {
-	var toRemoveMessageIndexes []int
+func appendToSet(set map[string]struct{}, elements []string) {
```

Comment:
Rename this to addToSet? Set and append don't go well together.

Reply:
Done.